### DISCUSSION TITLE
#94 - Ripples 91% match issue using borders

### DISCUSSION THREAD

**Discussion by sttrsttr** [cssbattle/forum#174] [Q&A] [answered]
I am struggeling to understand why this solution only recieves 91% match scoring. My diff is completelty black. If I zoom close I can se some diff but that is <1 px so I cannot do anything about it (?)

`&lt;p&gt;&lt;dl&gt;&lt;dm&gt;&lt;style&gt;
* {
    background: var(--,#0E2E2C);
    margin: 0;
    position: absolute;
    border-radius: 50%;
  }
 p {
    width: 160px;
    height: 160px;
    top: 56px;
    left: 106px;
    --: #F3AC3C;
    border: 14px #0E2E2C solid;
    outline: 6px #F3AC3C solid
  }
  dl {
    width: 100px;
    height: 100px;
    top: 90px;
    left: 140px;
    --: #0E2E2C;
    border: 10px #998235 solid;
    outline: 12px #0E2E2C solid
  }  dm {
    width: 40px;
    height: 40px;
    --: #F3AC3C;
    left: 22;
    top: 22;
    border: 8px #0E2E2C solid;
    outline: 12px #F3AC3C solid`

**Comment by SelenIT** [comment, marked as answer]
It seems that the game backend uses the older Chromium version that doesn't support rounded corners of `outline`s, so, for the backend renderer, your solution looks like this:
<img width="403" alt="Screenshot 2021-11-15 at 00 26 48" src="https://user-images.githubusercontent.com/282754/141699320-09bab5d6-182f-426b-8b94-ab6ad57e7999.png">
(you can check it with the great @alexzaworski's [preview tool](https://codepen.io/alexzaworski/pen/vwmNKr))

The easiest fix would be replacing `outline` with `box-shadow` with zero offsets and needed spread radius.  However, maybe a different approach would be more efficient, like nesting elements instead of stacking them and using just `margin` to create dark circles between their borders?

**Reply by sttrsttr** [reply]
Thank you very much :) 

I knew it was not the most sexy solution, but still couldn't understand why the bad score.

Great preview tool
